cluster/src/main/java/com/hartwig/pipeline/resource/Resource.java

```diff
@@ -1,60 +1,27 @@
 package com.hartwig.pipeline.resource;
 
-import java.util.function.Function;
+import static com.hartwig.pipeline.resource.ResourceNames.COSMIC;
+import static com.hartwig.pipeline.resource.ResourceNames.DBSNPS;
+import static com.hartwig.pipeline.resource.ResourceNames.GC_PROFILE;
+import static com.hartwig.pipeline.resource.ResourceNames.REFERENCE_GENOME;
+import static com.hartwig.pipeline.resource.ResourceNames.SNPEFF;
 
-import com.google.api.gax.paging.Page;
-import com.google.cloud.storage.Blob;
-import com.google.cloud.storage.Bucket;
-import com.google.cloud.storage.Storage;
-import com.hartwig.pipeline.storage.RuntimeBucket;
+import com.hartwig.pipeline.execution.vm.VmDirectories;
 
-import org.slf4j.Logger;
-import org.slf4j.LoggerFactory;
+public interface Resource {
 
-public class Resource {
-
-    private static final Logger LOGGER = LoggerFactory.getLogger(Resource.class);
-
-    private final Storage storage;
-    private final String commonResourcesBucket;
-    private final String name;
-    private final Function<String, String> alias;
-
-    public Resource(final Storage storage, final String commonResourcesBucket, final String name, final Function<String, String> alias) {
-        this.storage = storage;
-        this.commonResourcesBucket = commonResourcesBucket;
-        this.name = name;
-        this.alias = alias;
+    static String of(String name, String file) {
+        return String.format("%s/%s/%s", VmDirectories.RESOURCES, name, file);
     }
 
-    public Resource(final Storage storage, final String commonResourcesBucket, final String name) {
-        this(storage, commonResourcesBucket, name, Function.identity());
+    static String of(String name) {
```

Comment:
Would it be worth calling this `dir` or something similar (as opposed to a file)?